<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIContactTieDialog</class>
 <widget class="QWidget" name="GUIContactTieDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contact Tie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>Tie Nameï¼š</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="lineEdit_name">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_matID_2">
     <property name="text">
      <string>Tie KeyWord:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_KeyWord">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Main Surface</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Secondary Node Group</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Ignore Secondary Nodes</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBox_IgnoreSecond"/>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Spotweld Formulation</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QComboBox" name="comboBox_SpotwldForm"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Hierarchy Level</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="lineEdit_HL"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Search Formulation</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QComboBox" name="comboBox_SearchForm"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Node Deletion</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QComboBox" name="comboBox_NodeDeletion"/>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Search Distance</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="lineEdit_SearchDistance"/>
   </item>
   <item row="10" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <widget class="GUI::NodeGroupWidget" name="widget_MainSurface" native="true"/>
   </item>
   <item row="3" column="3">
    <widget class="GUI::NodeGroupWidget" name="widget_SecNode" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GUI::NodeGroupWidget</class>
   <extends>QWidget</extends>
   <header>NodeGroupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
